public/locales/en/translation.json

```diff
@@ -1,959 +1,966 @@
-{

```

Comment:
all these changes are because I ran `yarn i18n`

Reply:
You probably edited the file manually and then saved and it changed the formatting. We should never edit the json file from translations directly, always use the i18n command when adding or removing new keys.